packages/patternfly-4/react-core/src/components/Alert/Alert.js

```diff
@@ -60,20 +83,32 @@ const Alert = ({
     </React.Fragment>
   );
 
-  const customClassName = css(styles.alert, getModifier(styles, variant, styles.modifiers.info), className);
-
+  const customClassName = css(
+    styles.alert,
+    getModifier(styles, variant, styles.modifiers.info),
+    className,
+    ((toast || ariaLive) && 'pf-m-live')
```

Comment:
I don't think we want `ariaLive` here. The purpose of `pf-m-live` is that it will enable us to add css to core that will add the delay that some screen readers need in perceiving the contents.

And the purpose of `toast` is to use the Alert component with all the things we recommend to make the alert accessible, following our design patterns. If the consumer doesn't want to use `toast` and instead wants to manually define aria attributes for the live region, then they should also manually add the class `pf-m-live` that handles that delay.